<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyhoteeMainWindow</class>
 <widget class="QMainWindow" name="KeyhoteeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyhotee</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="fixed_tool_bar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="new_message_button">
         <property name="text">
          <string>M</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>889</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="new_contact_button">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="side_bar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>220</red>
             <green>225</green>
             <blue>232</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>237</red>
             <green>237</green>
             <blue>237</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="indentation">
        <number>13</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>IDENTITIES</string>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>91</red>
           <green>91</green>
           <blue>91</blue>
          </color>
         </brush>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MAILBOXES</string>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>91</red>
           <green>91</green>
           <blue>91</blue>
          </color>
         </brush>
        </property>
        <item>
         <property name="text">
          <string>Inbox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drafts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sent</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CONTACTS</string>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>127</red>
           <green>127</green>
           <blue>127</blue>
          </color>
         </brush>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>91</red>
           <green>91</green>
           <blue>91</blue>
          </color>
         </brush>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
      <widget class="QStackedWidget" name="widget_stack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>40</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_2"/>
       </widget>
       <widget class="QWidget" name="contacts_page">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="ContactsTable" name="widget" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMail">
    <property name="title">
     <string>Mail</string>
    </property>
    <addaction name="actionNew_Message"/>
   </widget>
   <widget class="QMenu" name="menuContact">
    <property name="title">
     <string>Contact</string>
    </property>
    <addaction name="actionNew_Contact"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuMail"/>
   <addaction name="menuContact"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMail">
   <property name="text">
    <string>Mail</string>
   </property>
  </action>
  <action name="actionNew_Contact">
   <property name="text">
    <string>New Contact</string>
   </property>
  </action>
  <action name="actionNew_Message">
   <property name="text">
    <string>New Message</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactsTable</class>
   <extends>QWidget</extends>
   <header>AddressBook/ContactsTable.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Keyhotee.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>new_message_button</sender>
   <signal>clicked()</signal>
   <receiver>actionNew_Message</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>new_contact_button</sender>
   <signal>clicked()</signal>
   <receiver>actionNew_Contact</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>949</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
